tests/KdybyTests/Aop/Extension.phpt

```diff
@@ -40,7 +40,6 @@ class ExtensionTest extends Tester\TestCase
 	{
 		$config = new Nette\Configurator();
 		$config->setTempDirectory(TEMP_DIR);
-		$config->addParameters(array('container' => array('class' => 'SystemContainer_' . md5($configFile . microtime()))));
```

Comment:
With 2.2 it's ok. In 2.3 there is a new name convention for system container (like `Container_05082f636f`).